src/feature/tests/__snapshots__/countryInfo.test.js.snap

```diff
@@ -0,0 +1,106 @@
+      <p>
+        Contry details
+      </p>
+      <div
+        className="header-icons"
+      >
+        <svg
+          aria-hidden="true"
+          className="svg-inline--fa fa-microphone fa-w-11 fa-lg header-icon"
+          color="#fff"
+          data-icon="microphone"
+          data-prefix="fas"
+          focusable="false"
+          role="img"
+          style={Object {}}
+          viewBox="0 0 352 512"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M176 352c53.02 0 96-42.98 96-96V96c0-53.02-42.98-96-96-96S80 42.98 80 96v160c0 53.02 42.98 96 96 96zm160-160h-16c-8.84 0-16 7.16-16 16v48c0 74.8-64.49 134.82-140.79 127.38C96.71 376.89 48 317.11 48 250.3V208c0-8.84-7.16-16-16-16H16c-8.84 0-16 7.16-16 16v40.16c0 89.64 63.97 169.55 152 181.69V464H96c-8.84 0-16 7.16-16 16v16c0 8.84 7.16 16 16 16h160c8.84 0 16-7.16 16-16v-16c0-8.84-7.16-16-16-16h-56v-33.77C285.71 418.47 352 344.9 352 256v-48c0-8.84-7.16-16-16-16z"
+            fill="currentColor"
+            style={Object {}}
+          />
+        </svg>
+        <svg
+          aria-hidden="true"
+          className="svg-inline--fa fa-cog fa-w-16 fa-lg header-icon"
+          color="#fff"
+          data-icon="cog"
+          data-prefix="fas"
+          focusable="false"
+          role="img"
+          style={Object {}}
+          viewBox="0 0 512 512"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z"
+            fill="currentColor"
+            style={Object {}}
+          />
+        </svg>
+      </div>
+    </div>
+  </div>
+  <div
+    className="country-details"
+  >
+    <h2 />
+    <p>
+      Source:
+       
+    </p>
+    <p>
+      Deaths until today:
+       
+    </p>
+    <p>
+      Deaths today:
+       
+    </p>
+    <p>
+      Confirmed cases until today:
+       
+    </p>
+    <p>
+      Confirmed cases today:
+       
+    </p>
+    <p>
+      Open cases until today:
+       
+    </p>
+    <p>
+      Recovered cases until today:
+       
+    </p>
+    <p>
+      New open cases:
+       
+    </p>
+    <p>
+      New recovered cases:
+       
+    </p>
+  </div>
+</div>
+`;
```

Comment:
Thank you for adding tests to your project. 

- [ ] It would be great if you can resolve all the failing tests. 
<img width="603" alt="Capture" src="https://user-images.githubusercontent.com/58504655/133412315-15c2ca0e-2939-4820-a305-671a053088ba.PNG">

Reply:
Tests fixed.